### PR TITLE
feat: list component

### PR DESCRIPTION
**Pull request by ravikumar0403**
Change log:
Added different variants of list components
- Unordered List
- Ordered List
- List with no bullets
- Inline list
- Stacked List
image
image
image
image
image

### CODE REVIEW

File: css/list.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
+.list{
+    padding: 0;
+    padding-left: 1rem;
```
Comment (NeerajVodala) [inline css/list.css]: can use shorthand property and finish it in single line.

File: css/list.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    display: flex;
+    list-style-type: none;
+    padding: 0;
+    gap: 1rem;
+}
+
+.list.stacked{
+    list-style-type: none;
+    padding: 0;
+}
```
Comment (NeerajVodala) [inline css/list.css]: /* list-style-type: none;
padding: 0; */

can be avoided (code same as .no-bullets [line-42] )

**Review by NeerajVodala — COMMENTED** [review]
Recommended changes (not mandatory): Added comments in list.css file 

Rest everything looks fine. Best of luck.

File: index.html [hunk 3]
```diff
@@ -791,7 +792,196 @@ <h2>Disabled and Read Only input</h2>
...
+                    <ol class="list ol-number" reversed>
+                        <li>Reversed list item</li>
+                        <li>Reversed list item</li>
+                        <li>Reversed list item</li>
+                        <li>Reversed list item</li>
+                    </ol>
+                </div>
+                <pre>
+                    <code class="html">
+&lt;ol class="list ol-number"&gt;
```
Comment (chanchal16) [inline index.html]: indent it properly. To remove the white spaces from the code block you can style the code tag either by giving white-space: pre-line/pre-wrap or by giving negative left margin.

Comment (ravikumar0403) [reply index.html]: Thank you @chanchal16. I wasn't aware of this property. Will make the change

**Pull request merged**